### PR TITLE
Holo projectors QOL

### PR DESCRIPTION
**Pull request by CydiaLamiales**
## About The Pull Request
Adds the maxium holosign amount and current amount of active holosigns to the holo projector description, also adds text saying how many sighs are left after each successful projection.

## Why It's Good For The Game
How many signs each projector type can hold can only be found out by placing the signs untill the limit then counting them. Such obscurity is not good for the game. Also allows people to check their projector and see if they maybe have an active sign or two across the station that they're forgotten about. 

 
## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

holofans

</details>

## Changelog
:cl:
add: Added the maximum sigh capacity and current amount of holo signs active to the description of holo projectors.
add: Adds a text saying how many holo signs are left after each successful holo sign placement.
/:cl:

### CODE REVIEW

**Comment by Tsar-Salat** [comment]
epic
much needed change

File: code/game/objects/items/holosign_creator.dm [hunk 1]
```diff
@@ -33,36 +34,53 @@
...
+						if(length(signs) >= max_signs)
 							return
 						if(is_blocked_turf(T, TRUE)) //don't try to sneak dense stuff on our tile during the wait.
 							return
 					H = new holosign_type(get_turf(target), src)
-					to_chat(user, "<span class='notice'>You create \a [H] with [src].</span>")
+					if(length(signs) == max_signs)
+						to_chat(user, "<span class='notice'>You create \a [H] with [src]. It cannot project any more [sign_name]\s!</span>")
+					else
+						to_chat(user, "<span class='notice'>You create \a [H] with [src]. It can project [max_signs - length(signs)] more [sign_name]\s</span>")
```
Comment (PowerfulBacon) [inline code/game/objects/items/holosign_creator.dm]: Add a fullstop at the end

File: code/game/objects/items/holosign_creator.dm [hunk 2]
```diff
@@ -33,36 +34,53 @@
...
+	. = ..()
+	. += "It has a maximum capacity of [max_signs] [sign_name]\s"
+	if(!length(signs))
+		. += "It is currently not projecting any [sign_name]\s."
+		return
+	if(length(signs) < max_signs)
+		. += "It is currently projecting [length(signs)] [sign_name]\s."
+		return
+	if(length(signs) == max_signs)
+		. += "It is currently projecting at maximum capacity!."
```
Comment (PowerfulBacon) [inline code/game/objects/items/holosign_creator.dm]: Double punctuation

File: code/game/objects/items/holosign_creator.dm (hunk 1 above)
Comment (CydiaLamiales) [reply code/game/objects/items/holosign_creator.dm]: What do you mean by 'fullstop', a dot?

File: code/game/objects/items/holosign_creator.dm [hunk 3]
```diff
@@ -96,45 +117,49 @@
 	desc = "A holographic projector that creates PENLITE holobarriers. Useful during quarantines since they halt those with malicious diseases."
 	icon_state = "signmaker_med"
 	holosign_type = /obj/structure/holosign/barrier/medical
+	sign_name = "holobarrier"
 	creation_time = 30
 	max_signs = 3
 
 /obj/item/holosign_creator/cyborg
-	name = "Energy Barrier Projector"
+	name = "\improper Energy Barrier Projector"
```
Comment (itsmeow) [inline code/game/objects/items/holosign_creator.dm]: if you could just remove the improper and uncapitalize this

**Review by itsmeow — CHANGES_REQUESTED** [review]

File: code/game/objects/items/holosign_creator.dm (hunk 3 above)
Comment (CydiaLamiales) [reply code/game/objects/items/holosign_creator.dm]: done

**Review by itsmeow — APPROVED** [review]

**Comment by Tsar-Salat** [comment]
nice

**Pull request merged**